projects/lib/src/lib/components/logout/logout.component.ts

```diff
@@ -22,7 +22,7 @@ export class LogoutComponent implements OnInit {
     private luigiCoreService: LuigiCoreService,
     private ref: ChangeDetectorRef,
     private i18nService: I18nService,
-    private loginEventService: LoginEventService
+    private loginEventService: LoginEventService,
```

Comment:
you can additionally make this comonent the OnPush and remove the `this.ref.detectChanges();` and use the signals